.github/utils/CheckExamples/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# This just checks that files in the Check folders are compilable to avoid
+# committing incorrect examples. It's done here, separately and in bulk, so that
+# no overhead is added by default when submitting a new check. You'll hopefully
+# only have to know this exists if your example fails to compile
+
+cmake_minimum_required(VERSION 3.25)
+project("Test that the check examples compile")
+
+enable_language(Fortran OPTIONAL)
+# Build them as libraries, not executables
+set (CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+set (BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../Checks)
+message(STATUS "Looking for files in '${BASE_DIR}'")
+
+file(GLOB check_files
+  ${BASE_DIR}/*/*.c
+  ${BASE_DIR}/*/*.cpp
+  ${BASE_DIR}/*/*.f
+  ${BASE_DIR}/*/*.f90
+)
```

Comment:
See existing Actions for additional file extensions: https://github.com/codee-com/open-catalog/blob/c66afdf420f5c67824c83da14754b0e39e24d536/.github/workflows/test-benchmarks.yml#L9-L15

I'd also add a comment to clarify that this excludes benchmark subdirectories within `Checks/`, such as: https://github.com/codee-com/open-catalog/tree/c66afdf420f5c67824c83da14754b0e39e24d536/Checks/PWR072/benchmark

Reply:
Yep, done